docs/native-numbers.md

```diff
@@ -50,13 +50,22 @@ minus_zero = -0.0
 ### Mixing number types
 
 The three number types are fundamentally different.
-If you mix two numbers of different types, HVM will interpret the binary representation of one of them incorrectly, leading to incorrect results. Which number is interpreted incorrectly depends on the situation and shouldn't be relied on for now.
+If you mix two numbers of different types, HVM will interpret the binary representation of one of them incorrectly, leading to incorrect results. Which number is interpreted incorrectly depends on the situation and shouldn't be relied on for now. Instead, you should make sure that all numbers are of the same type. We recently introduced a way to convert between the different number types, and using it is very easy, here's an example:
```

Comment:
Don't say "we recently introduced" since this document will someday get old. Also separate that into a different section/paragraph